lib/ssl/src/dtls_record.erl

```diff
@@ -433,10 +421,11 @@ get_dtls_records_aux({DataTag, StateName, _, Versions} = Vinfo,
         orelse ((StateName == abbreviated) andalso (DataTag == udp)))
        andalso ((Type == ?HANDSHAKE) orelse (Type == ?ALERT)) ->
     ssl_logger:debug(LogLevel, inbound, 'record', [RawDTLSRecord]),
-    Acc = [#ssl_tls{type = Type, version = {MajVer, MinVer},
+    Version = {MajVer,MinVer},
```

Comment:
@KennethL I do not think we have any fundamental disagreement, my point is that we should do incremental improvements.  And then it will also be easier to review and evaluate the improvement ideas in a bigger perspective. I think the internal representation should be step 2. First we only do the macros!

Reply:
Yes the first step is to use macros all over the place and my point here is to use the variable name `Version` when it is a value representation which we have defined macros for. And call it something else, for example `Version_raw` or `Version_on_wire` when it holds a representation of version that should be sent or received on the wire.